src/SampleWebApp/Controllers/AuthorsMvcController.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    public class AuthorsMvcController : Controller
+    {
+        private AuthorDataContext _context;
+
+        public AuthorsMvcController(AuthorDataContext context)
+        {
+            _context = context;    
+        }
+
+        // GET: AuthorsMvc
+        public IActionResult Index()
+        {
+            return View(_context.Author.ToList());
```

Comment:
I had commented on the API one first, but I'll repeat what I said there... I'm not sure there's any reason to default to sync - when we were doing this before, async was a *new* feature and plenty of people didn't grok it. Now it seems like async should be the default.